rabbitmq-smoke-tests-dev

```diff
@@ -216,7 +216,7 @@ fi
 bosh_target_check
 bosh_cli_check
 
-declare -a args
+declare -a args=(args)
```

Comment:
If I am reading this correctly, you are setting the initial argument list to ("args"), and then, if the caller specified arguments on the CLI, we override them (line 221).

Given the check of `$# > 0` on line 207 (above), I don't *think* we'll ever run afoul of a bug where we get a fake "args" argument, but can we side-step the issue entirely by just setting `args=` and letting line 221 turn it into an array?

The following test script seems to work in all cases:

```
#!/bin/bash
set -u
args=
if (( ${#@} ))
  then args=($(echo "${@}"))
fi
for x in "${args[@]}"; do
  echo "found arg [$x]"
done
```